### PR TITLE
Remove Deprecated `Id<A>` data type

### PR DESCRIPTION
**Pull request by danimontoya**

### CODE REVIEW

File: arrow-fx/src/test/kotlin/arrow/fx/ScheduleTest.kt [hunk 1]
```diff
@@ -191,49 +78,6 @@ class ScheduleTest : ArrowFxSpec() {
...
-        val sum = res.fold(0L) { acc, v ->
-          acc + v.delay.amount
-        }
-        val fibSum = fibs(i.toLong()).drop(1).take(res.size).sum()
-
-        res.forAll { it.cont } && sum == fibSum
-      }
-    }
-
-    "Schedule.linear()" {
```
Comment (i-walker) [inline arrow-fx/src/test/kotlin/arrow/fx/ScheduleTest.kt:215]: Are those to be replaced, with something else, or is this not required anymore? 
cc @nomisRev

Comment (nomisRev) [reply arrow-fx/src/test/kotlin/arrow/fx/ScheduleTest.kt:215]: Arrow Fx is deprecated and will be removed in 0.13.0 so this doesn't need to be replaced.

**Review by nomisRev — APPROVED** [review]
Thanks @danimontoya

File: arrow-fx/src/test/kotlin/arrow/fx/ScheduleTest.kt (hunk 1 above)
Comment (danimontoya) [reply arrow-fx/src/test/kotlin/arrow/fx/ScheduleTest.kt:215]: thanks I was asking the same myself 👍

**Review by i-walker — APPROVED** [review]

**Pull request merged**